DigitalLearningSolutions.Data/Models/DetailedProgress.cs

```diff
@@ -0,0 +1,73 @@
+﻿namespace DigitalLearningSolutions.Data.Models
+{
+    using System;
+    using System.Collections.Generic;
+    using DigitalLearningSolutions.Data.Models.Courses;
+
+    public class DetailedCourseProgress
+    {
+        public DetailedCourseProgress(Progress progress, IEnumerable<DetailedSectionProgress> sections, DelegateCourseInfo delegateCourseInfo)
+        {
+            DiagnosticScore = progress.DiagnosticScore;
+            ProgressId = progress.ProgressId;
+            CustomisationId = progress.CustomisationId;
+            DelegateId = progress.CandidateId;
+
+            DelegateName = delegateCourseInfo.DelegateFirstName == null
+                ? delegateCourseInfo.DelegateLastName
+                : $"{delegateCourseInfo.DelegateFirstName} {delegateCourseInfo.DelegateLastName}";

```

Comment:
We have a helper that does this delegate name string setup. I think we should pass up both of these properties to the view model, then convert to the full name using that there.